Makefile

```diff
@@ -32,3 +33,5 @@ clean:
 distclean: clean
 	$(RM) -r $(DCURL_DIR)
 	git checkout $(DCURL_DIR)
+
+include mk/python.mk
```

Comment:
Don't include here. Instead, move to earlier in order to prevent incorrect Python and PyOTA installation.